test/accounts.js

```diff
@@ -1,263 +1,178 @@
-var BN = require("bn.js");
-var Web3 = require("web3");
-var Ganache = require(process.env.TEST_BUILD
-  ? "../build/ganache.core." + process.env.TEST_BUILD + ".js"
-  : "../index.js");
-var assert = require("assert");
+const assert = require("assert");
+const initializeTestProvider = require("./helpers/web3/initializeTestProvider");
 
-describe("Accounts", function() {
-  var expectedAddress = "0x604a95C9165Bc95aE016a5299dd7d400dDDBEa9A";
-  var mnemonic = "into trim cross then helmet popular suit hammer cart shrug oval student";
+describe("Accounts", async() => {
```

Comment:
I can't remember why, but I think you said we should be converting to arrow functions on `describe` callbacks. Does that apply here?

Reply:
We should definitely avoid arrow functions with hooks such as `before`, `after`, etc...  because it unbinds `this` (aka Mocha context) and prevents configuring timeouts for a block.